libs/core-ui/src/lib/util/buildInitialContext.ts

```diff
@@ -125,3 +126,10 @@ export function getModelType(
       return ModelTypes.Multiclass;
   }
 }
+
+export function ifEnableLargeData(dataset: IDataset): boolean {
+  if (dataset?.is_large_data_scenario && dataset?.use_entire_test_data) {
```

Comment:
nit: just return dataset?.is_large_data_scenario && dataset?.use_entire_test_data

Reply:
Looks like returning the ```dataset?.is_large_data_scenario && dataset?.use_entire_test_data;``` raises an error.

```
ERROR in libs/core-ui/src/lib/util/buildInitialContext.ts:131:3
TS2322: Type 'boolean | undefined' is not assignable to type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'.
    129 |
    130 | export function ifEnableLargeData(dataset: IDataset): boolean {
  > 131 |   return dataset?.is_large_data_scenario && dataset?.use_entire_test_data;
        |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    132 | }
    133 |
```

Reply:
Is there any other way to simplify this code?